portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,40 +32,43 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> messages;
   private static final String COMMENT = "comment-input";
   private static final String NAME = "name-input";
-
-  @Override
-  public void init() {
-    messages = new ArrayList<>();
-  }
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I think this can also probably be private static final, but I'm not sure if you'll get errors.

Reply:
I converted this to private static final, and this runs without any errors :)